blocks/library/button/test/__snapshots__/index.js.snap

```diff
@@ -7,12 +7,22 @@ exports[`core/button block edit matches snapshot 1`] = `
   <div
     class="blocks-rich-text"
   >
-    <span
-      aria-label="Add text…"
-      class="wp-block-button__link blocks-rich-text__tinymce"
-      contenteditable="true"
-      data-is-placeholder-visible="true"
-    />
+    <div>
```

Comment:
I noticed this as well and was initially puzzled, but they are divs added by the Autocomplete component now used within RichText:

* `Autocomplete` adds a div to wrap TinyMCE, adding event listeners and a sibling popup.
* `withFocusOutside` wraps `Autocomplete` and adds a parent div because it adds event listeners.
* The autocompleters hook adds another parent div in order to add its own event listeners.

I dislike this but do not currently see a better solution. I would love to be able to add event listeners to a Fragment (mentioned [here](https://reactjs.org/docs/fragments.html#keyed-fragments) as a possible future capability) to avoid rendering the elements. I'll give it some thought. Do you have any suggestions?